tools/windows/DatadogAgentInstaller/CustomActions/CustomActions.csproj

```diff
@@ -0,0 +1,92 @@
+    <Optimize>true</Optimize>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
+    <DebugSymbols>true</DebugSymbols>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DebugType>full</DebugType>
+    <PlatformTarget>x64</PlatformTarget>
+    <LangVersion>7.3</LangVersion>
+    <ErrorReport>prompt</ErrorReport>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
+    <DefineConstants>TRACE</DefineConstants>
+    <Optimize>true</Optimize>
+    <DebugType>pdbonly</DebugType>
+    <PlatformTarget>x64</PlatformTarget>
+    <LangVersion>7.3</LangVersion>
+    <ErrorReport>prompt</ErrorReport>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="CleanUpFilesCustomAction.cs" />
+    <Compile Include="CompilerExtensions.cs" />
+    <Compile Include="ConfigCustomActions.cs" />
+    <Compile Include="PythonDistributionCustomAction.cs" />
+    <Compile Include="Extensions\SessionExtensions.cs" />
+    <Compile Include="ISession.cs" />
+    <Compile Include="SessionWrapper.cs" />
+    <Compile Include="UserCustomActions.cs" />
+    <Compile Include="Native\NativeMethods.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup />
+  <ItemGroup>
+    <Reference Include="Cave.Compression, Version=2.1.0.0, Culture=neutral, PublicKeyToken=81fb9a2cd82ecf81, processorArchitecture=MSIL">
+      <HintPath>$(NuGetPackagesDir)\Cave.Compression.2.1.0\lib\net35\Cave.Compression.dll</HintPath>
+    </Reference>
+    <Reference Include="Cave.Extensions, Version=2.1.4.0, Culture=neutral, PublicKeyToken=65480c9a544d67e6, processorArchitecture=MSIL">
+      <HintPath>$(NuGetPackagesDir)\Cave.Extensions.2.1.4\lib\net35\Cave.Extensions.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Deployment.WindowsInstaller, Version=3.0.0.0, Culture=neutral, PublicKeyToken=ce35f76fcda82bad, processorArchitecture=MSIL" />
+    <Reference Include="SevenZip, Version=19.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>$(NuGetPackagesDir)\LZMA-SDK.19.0.0\lib\net20\SevenZip.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Configuration.Install" />
+    <Reference Include="System.DirectoryServices" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.ServiceProcess" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="YamlDotNet, Version=12.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e, processorArchitecture=MSIL">
+      <HintPath>$(NuGetPackagesDir)\YamlDotNet.12.0.0\lib\net35\YamlDotNet.dll</HintPath>
+    </Reference>
```

Comment:
@bkabrda How do we consider list of referenced DLLs for SBOM? Or other security related risks?